Microsoft Windows [Version 10.0.22631.4751]
(c) Microsoft Corporation. All rights reserved.

C:\Users\欧阳葭>wsl
Welcome to Ubuntu 24.04.2 LTS (GNU/Linux 5.15.167.4-microsoft-standard-WSL2 x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/pro

 System information as of Thu Feb 13 09:51:23 CST 2025

  System load:  0.08                Processes:             90
  Usage of /:   0.3% of 1006.85GB   Users logged in:       0
  Memory usage: 3%                  IPv4 address for eth0: 172.19.133.12
  Swap usage:   0%

 * Strictly confined Kubernetes makes edge and IoT secure. Learn how MicroK8s
   just raised the bar for easy, resilient and secure K8s cluster deployment.

   https://ubuntu.com/engage/secure-kubernetes-at-the-edge

This message is shown once a day. To disable it please create the
/home/suki/.hushlogin file.
(base) suki@LAPTOP-5PMP3V9O:/mnt/c/Users/欧阳葭$ clustalw -help



 CLUSTAL 2.1 Multiple Sequence Alignments


                DATA (sequences)

-INFILE=file.ext                             :input sequences.
-PROFILE1=file.ext  and  -PROFILE2=file.ext  :profiles (old alignment).


                VERBS (do things)

-OPTIONS            :list the command line parameters
-HELP  or -CHECK    :outline the command line params.
-FULLHELP           :output full help content.
-ALIGN              :do full multiple alignment.
-TREE               :calculate NJ tree.
-PIM                :output percent identity matrix (while calculating the tree)
-BOOTSTRAP(=n)      :bootstrap a NJ tree (n= number of bootstraps; def. = 1000).
-CONVERT            :output the input sequences in a different file format.


                PARAMETERS (set things)

***General settings:****
-INTERACTIVE :read command line, then enter normal interactive menus
-QUICKTREE   :use FAST algorithm for the alignment guide tree
-TYPE=       :PROTEIN or DNA sequences
-NEGATIVE    :protein alignment with negative values in matrix
-OUTFILE=    :sequence alignment file name
-OUTPUT=     :CLUSTAL(default), GCG, GDE, PHYLIP, PIR, NEXUS and FASTA
-OUTORDER=   :INPUT or ALIGNED
-CASE        :LOWER or UPPER (for GDE output only)
-SEQNOS=     :OFF or ON (for Clustal output only)
-SEQNO_RANGE=:OFF or ON (NEW: for all output formats)
-RANGE=m,n   :sequence range to write starting m to m+n
-MAXSEQLEN=n :maximum allowed input sequence length
-QUIET       :Reduce console output to minimum
-STATS=      :Log some alignents statistics to file

***Fast Pairwise Alignments:***
-KTUPLE=n    :word size
-TOPDIAGS=n  :number of best diags.
-WINDOW=n    :window around best diags.
-PAIRGAP=n   :gap penalty
-SCORE       :PERCENT or ABSOLUTE


***Slow Pairwise Alignments:***
-PWMATRIX=    :Protein weight matrix=BLOSUM, PAM, GONNET, ID or filename
-PWDNAMATRIX= :DNA weight matrix=IUB, CLUSTALW or filename
-PWGAPOPEN=f  :gap opening penalty
-PWGAPEXT=f   :gap opening penalty


***Multiple Alignments:***
-NEWTREE=      :file for new guide tree
-USETREE=      :file for old guide tree
-MATRIX=       :Protein weight matrix=BLOSUM, PAM, GONNET, ID or filename
-DNAMATRIX=    :DNA weight matrix=IUB, CLUSTALW or filename
-GAPOPEN=f     :gap opening penalty
-GAPEXT=f      :gap extension penalty
-ENDGAPS       :no end gap separation pen.
-GAPDIST=n     :gap separation pen. range
-NOPGAP        :residue-specific gaps off
-NOHGAP        :hydrophilic gaps off
-HGAPRESIDUES= :list hydrophilic res.
-MAXDIV=n      :% ident. for delay
-TYPE=         :PROTEIN or DNA
-TRANSWEIGHT=f :transitions weighting
-ITERATION=    :NONE or TREE or ALIGNMENT
-NUMITER=n     :maximum number of iterations to perform
-NOWEIGHTS     :disable sequence weighting


***Profile Alignments:***
-PROFILE      :Merge two alignments by profile alignment
-NEWTREE1=    :file for new guide tree for profile1
-NEWTREE2=    :file for new guide tree for profile2
-USETREE1=    :file for old guide tree for profile1
-USETREE2=    :file for old guide tree for profile2


***Sequence to Profile Alignments:***
-SEQUENCES   :Sequentially add profile2 sequences to profile1 alignment
-NEWTREE=    :file for new guide tree
-USETREE=    :file for old guide tree


***Structure Alignments:***
-NOSECSTR1     :do not use secondary structure-gap penalty mask for profile 1
-NOSECSTR2     :do not use secondary structure-gap penalty mask for profile 2
-SECSTROUT=STRUCTURE or MASK or BOTH or NONE   :output in alignment file
-HELIXGAP=n    :gap penalty for helix core residues
-STRANDGAP=n   :gap penalty for strand core residues
-LOOPGAP=n     :gap penalty for loop regions
-TERMINALGAP=n :gap penalty for structure termini
-HELIXENDIN=n  :number of residues inside helix to be treated as terminal
-HELIXENDOUT=n :number of residues outside helix to be treated as terminal
-STRANDENDIN=n :number of residues inside strand to be treated as terminal
-STRANDENDOUT=n:number of residues outside strand to be treated as terminal


***Trees:***
-OUTPUTTREE=nj OR phylip OR dist OR nexus
-SEED=n        :seed number for bootstraps.
-KIMURA        :use Kimura's correction.
-TOSSGAPS      :ignore positions with gaps.
-BOOTLABELS=node OR branch :position of bootstrap values in tree display
-CLUSTERING=   :NJ or UPGMA
(base) suki@LAPTOP-5PMP3V9O:/mnt/c/Users/欧阳葭$ /home/suki/phylogenetics-class/primatesAA.fasta
-bash: /home/suki/phylogenetics-class/primatesAA.fasta: Permission denied
(base) suki@LAPTOP-5PMP3V9O:/mnt/c/Users/欧阳葭$ ls -l /home/suki/phylogenetics-class/primatesAA.fasta
-rw-r--r-- 1 suki suki 11435 Feb 11 14:32 /home/suki/phylogenetics-class/primatesAA.fasta
(base) suki@LAPTOP-5PMP3V9O:/mnt/c/Users/欧阳葭$ cat /home/suki/phylogenetics-class/primatesAA.fasta
>Human
MASGILVNVKEEVTCPICLELLTQPLSLDCGHSFCQACLTANHKKSMLDKGESSCPVCRISYQPENIRPNRHVANIVEKL
REVKLSPEGQKVDHCARHGEKLLLFCQEDGKVICWLCERSQEHRGHHTFLTEEVAREYQVKLQAALEMLRQKQQEAEELE
ADIREEKASWKTQIQYDKTNVLADFEQLRDILDWEESNELQNLEKEEEDILKSLTNSETEMVQQTQSLRELISDLEHRLQ
GSVMELLQGVDGVIKRTENVTLKKPETFPKNQRRVFRAPDLKGMLEVFRELTDVRRYWVDVTVAPNNISCAVISEDKRQV
SSPKPQIIYGARGTRYQTFVNFNYCTGILGSQSITSGKHYWEVDVSKKTAWILGVCAGFQPDAMCNIEKNENYQPKYGYW
VIGLEEGVKCSAFQDSSFHTPSVPFIVPLSVIICPDRVGVFLDYEACTVSFFNITNHGFLIYKFSHCSFSQPVFPYLNPR
KCGVPMTLCSPSS
>Chimp
MASGILVNVKEEVTCPICLELLTQPLSLDCGHSFCQACLTANHKKSMLDKGESSCPVCRISYQPENIRPNRHVANIVEKL
REVKLSPEGQKVDHCAHHGEKLLLFCQEDGKVICWLCERSQEHRGHHTFLTEEVAREYQVKLQAALEMLRQKQQEAEELE
ADIREEKASWKTQIQYDKTNVLADFEQLRDILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQSVRELISDLERRLQ
GSVMELLQGVDGVIKRMENVTLKKPETFPKNQRRVFRAPDLKGMLEVFRELTDVRRYWVDVTVAPNNISCAVISEDMRQV
SSPKPQIIYGARGTRYQTFMNFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMCNIEKNENYQPKYGYW
VIGLEEGVKCSAFQDGSFHTPSAPFIVPLSVIICPDRVGVFLDYEACTVSFFNITNHGSLIYKFSHCSFSQPVFPYLNPR
KCGVPMTLCSPSS
>Gorilla
MASGILVNVKEEVTCPICLELLTQPLSLDCGHSFCQACLTANHKKSMLDKGESSCPVCRISYQPENIRPNRHVANIVEKL
REVKLSPEGQKVDHCARHGEKLLLFCQEDGKVICWLCERSQEHRGHHTFLTEEVAQEYQVKLQAALEMLRQKQQEAEELE
ADIREEKASWKTQIQYDKTNVLADFEQLRDILDWEESNELQNLEKEEEDILKRLTKSETEMVQQTQSVRELISDLEHRLQ
GSVMELLQGVDGVIKRMENVTLKKPETFPKNRRRVFRAPDLKGMLEVFRELTDVRRYWVDVTVAPNNISCAVISEDMRQV
SSPKPQIIYGAQGTRYQTFMNFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDATCNIEKNENYQPKYGYW
VIGLEEGVKCSAFQDGSFHTPSAPFIVPLSVIICPDRVGVFLDYEACTVSFFNITNHGFLIYKFSHCSFSQPVFPYLNPR
KCRVPMTLCSPSS
>Orangutan
MASGILVNVKEEVTCPICLELLTQPLSLDCGHSFCQACLTANHKKSTLDKGERSCPVCRVSYQPKNIRPNRHVANIVEKL
REVKLSPEGQKVDHCARHGEKLLLFCKEDGKVICWLCERSQEHRGHHTFLTEEVAQKYQVKLQAALEMLRQKQQEAEELE
ADIREEKASWKTQIQYDKTSVLADFEQLRDILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQSVRELISDVEHRLQ
GSVMELLQGVDGIIKRMQNVTLKKPETFPKNQRRVFRAPNLKGMLEVFRELTDVRRYWVDVTVAPNDISYAVISEDMRQV
SCPEPQIIYGAQGTTYQTYVNFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYNIEQNENYQPQYGYW
VIGLEEGVKCSAFQDGSFHNPSAPFIVPLSVIICPDRVGVFLDYEACTVSFFNITNHGFLIYKFSHCSFSQPVFPYLNPR
KCRVPMTLCSPSS
>Gibbon
MASGILVNVKEKVTCPICLELLTQPLSLDCGHSFCQACLTANHKTSMPDEGERSCPVCRISYQHKNIRPNRHVANIVEKL
REVKLSPEEGQKVDHCARHGKKLLLFCQEDRKVICWLCERSQEHRGHHTFLTEEVAQEYQMKLQAALQMLRQKQQEAEEL
EADIREEKASWKTQIQYDKTNILADFEQLRHILDWVESNELQNLEKEEKDVLKRLMRSEIEMVQQTQSVRELISDLEHRL
QGSVMELLQGVDGVIKRMKNVTLKKPETFPKNRRRVFRAADLKVMLEVLRELRDVRRYWVDVTVAPNNISYAVISEDMRQ
VSSPEPQIIFEAQGTISQTFVNFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGLQPDAMYNIEQNENYQPKYGY
WVIGLEEGVKCNAFQDGSIHTPSAPFVVPLSVNICPDRVGVFLDYEACTVSFFNITDHGFLIYKFSHCSFSQPVFPYLNP
RKCTVPMTLCSPSS
>Rhes_cDNA
MASGILLNVKEEVTCPICLELLTEPLSLHCGHSFCQACITANHKKSMLYKEGERSCPVCRISYQPENIQPNRHVANIVEK
LREVKLSPEEGQKVDHCARHGEKLLLFCQEDSKVICWLCERSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEK
LEADIREEKASWKIQIDYDKTNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISELEHR
LQGSMMDLLQGVDGIIKRIENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDARRYWVDVTLATNNISHAVIAEDKR
QVSSRNPQIMYQAPGTLFTFPSLTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQSDAMYNIEQNENYQPK
YGYWVIGLQEGVKYSVFQDGSSHTPFAPFIVPLSVIICPDRVGVFVDYEACTVSFFNITNHGFLIYKFSQCSFSKPVFPY
LNPRKCTVPMTLCSPSS
>Baboon
MASGILLNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHRKSMLYKEGERSCPVCRISYQPENIQPNRHVANIVEK
LREVKLSPEEGLKVDHCARHGEKLLLFCQEDSKVICWLCERSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEK
LEADIREEKASWKIQIDYDKTNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDVRRYWVDVTLAPNNISHAVIAEDKR
QVSSRNPQITYQAPGTLFSFPSLTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYNIEQNENYQPK
YGYWVIGLQEGVKYSVFQDGSSHTPFAPFIVPLSVIICPDRVGVFVDYEACTVSFFNITNHGFLIYKFSQCSFSKPVFPY
LNPRKCTVPMTLCSPSS
>AGM
MASGILLNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHKESMLYKEEERSCPVCRISYQPENIQPNRHVANIVEK
LREVKLSPEEGQKVDHCARHGEKLLLFCQEDSKVICWLCERSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEK
LEADIREEKASWKIQIDYDKTNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDVRRYWVDVTLAPNNISHAVIAEDKR
QVSYQNPQIMYQAPGSSFGSLTNFNYCTGVLGSQSITSRKLTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAG
FQPDATYNIEQNENYQPKYGYWVIGLQEGDKYSVFQDSSSHTPFAPFIVPLSVIICPDRVGVFVDYEACTVSFFNITNHG
FLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>AGM_cDNA
MASGILVNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHKESMLYKEEERSCPVCRISYQPENIQPNRHVANIVEK
LREVKLSPEEGQKVDHCARHGEKLLLFCQEDSKVICWLCERSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEK
LEADIREEKASWKIQIDYDKTNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRVENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDVRRYWVDVTLAPNNISHAVIAEDKR
QVSYRNPQIMYQSPGSLFGSLTNFSYCTGVPGSQSITSGKLTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAG
FQPDATYNIEQNENYQPKYGYWVIGLQEGDKYSVFQDGSSHTPFAPFIVPLSVIICPDRVGVFVDYEACTVSFFNITNHG
FLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>Tant_cDNA
MASGILLNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHKESMLYKEEERSCPVCRISYQPENIQPNRHVANIVEK
LREVKLSPEEGQKVDHCARHGEKLLLFCQEDSKVICWLCERSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEK
LEADIREEKASWKIQIDYDKTNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDVRRYWVDVTLAPNNISHAVIAEDKR
QVSYQNPQIMYQAPGSSFGSLTNFNYCTGVLGSQSITSRKLTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAG
FQPDATYNIEQNENYQPKYGYWVIGLQEGDKYSVFQDGSSHTPFAPFIVPLSVIICPDRVGVFVDYEACTVSFFNITNHG
FLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>Patas
MASGILLNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHKKSMLYKEEERSCPVCRISYQPENIQPNRHVANIVEK
LREVKLSPEEGQKVDHCARHGEKLLLFCQEDRKVICWLCERSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEK
LEADIREEKASWKIQIDYDKTNVLADFEQLREILDWEESNELQYLEKEEEDILKSLTKSETKMVRQTQYVRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPETFHKNQRRVFRAPALKGMLDMFRELTDVRRYWVDVTLAPNNISHVVIAEDKR
QVSSRNPQIMYWAQGKLFQSLKNFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYDVEQNENYQPKYG
YWVIGLQEGVKYSVFQDGSSHTPFAPFIAPLSVIFCPDRVGVFVDYEACTVSFFNITNHGFLIYKFSQCSFSKPVFPYLN
PRKCTVPMTLCSPSS
>Colobus
MASGILVNIKEEVTCPICLELLTEPLSLHCGHSFCQACITANHKKSMLYKEGERSCPVCRISYQPENIRPNRHVANIVEK
LREVKLSPEEGQKVDHCARHGEKLLLFCQEDRKVICWLCERSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEK
LEADIREEKASWKIQIDYDKTNVLADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELVSDLEHR
LQGSVMELLQGVDGIIKRIEDMTLKKPKTFPKNQRRVFRAPDLKGMLDMFRELTDVRRYWVDVTLAPNNISHAVIAEDKR
RVSSPNPQIMYRAQGTLFQSLKNFIYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYNIEQNENYQPKYG
YWVIGLQEGVKYSVFQDGSSHTPFAPFIVPLSVIICPDRVGVFVDYEACTVSFFNITNHGFLIYKFSQCSFSKPVFPYLN
PRKCTVPMTLCSPSS
>DLangur
MASGILVNIKEEVTCPICLELLTEPLSLHCGHSFCQACITANHKKSMLYKEGERSCPVCRISYQPENIRPNRHVANIVEK
LREVKLSPEEGQKVDHCARHGEKLLLFCQEDRKVICWLCERSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEK
LEADIREEKASWKIQIDCDKTNVLADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPKTFPKNQRRVFRAPDLKGILDMFRELTDVRRYWVDVTLAPNNISHAVIAEDKR
QVSSPNPQIMCRARGTLFQSLKNFIYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYNIEQNENYQPKYG
YWVIGLQEGVKYNVFQDGSSHTPFAPFIVPLSVIICPDRVGVFVDYEACTVSFFNITNHGFLIYKFSQCSFSKPVFPYLN
PRKCTVPMTLCSPSS
>PMarmoset
MASRILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESTLHQGERSCPLCRMSYPSENLRPNRHLANIVERL
KEVMLSPEEGQKVDHCARHGEKLLLFCQQDGNVICWLCERSQEHRGHHTFLVEEVAEKYQGKLQVALEMMRQKQQDAEKL
EADVREEQASWKIQIQNDKTNIMAEFKQLRDILDCEESKELQNLEKEEKNILKRLVQSESDMVLQTQSIRVLISDLERRL
QGSVMELLQGVDDVIKRIEKVTLQKPKTFLNEKRRVFRAPDLKGMLQAFKELTEVQRYWAHVTLVPSHPSCTVISEDERQ
VRYQVPIHQPLVKVKYFYGVLGSLSITSGKHYWEVDVSNKRGWILGVCGSWKCNAKWNVLRPENYQPKNGYWVIGLRNTD
NYSAFQDAVKYSDVQDGSRSVSSGPLIVPLFMTICPNRVGVFLDYEACTISFFNVTSNGFLIYKFSNCHFSYPVFPYFSP
TTCELPMTLCSPSS
>Tamarin
MASRILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESTPHQGERSCPLCRMSYPSENLRPNRHLANIVERL
KEVMLSPEEGQKVGHCARHGEKLLLFCEQDGNVICWLCERSQEHRGHHTLLVEEVAEKYQEKLQVALEMMRQKQQDAEKL
EADVREEQASWKIQIRNDKTNIMAEFKQLRDILDCEESKELQNLEKEEKNILKRLVQSESDMVLQTQSMRVLISDLERRL
QGSVLELLQGVDDVIKRIETVTLQKPKTFLNEKRRVFRAPDLKAMLQAFKELTEVQRYWAHVTLVPSHPSYAVISEDERQ
VRYQFQIHQPSVKVNYFYGVLGSPSITSGKHYWEVDVTNKRDWILGICVSFKCNAKWNVLRPENYQPKNGYWVIGLQNTN
NYSAFQDAVKYSDFQIGSRSTASVPLIVPLFMTIYPNRVGVFLDYEACTVSFFNVTNNGFLIYKFSNCHFSYPVFPYFSP
MTCELPMTLCSPSS
>Squirrel
MASRILGSIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESMLHQGERSCPLCRLPYQSENLRPNRHLASIVERL
REVMLRPEERQNVDHCARHGEKLLLFCEQDGNIICWLCERSQEHRGHNTFLVEEVAQKYREKLQVALETMRQKQQDAEKL
EADVRQEQASWKIQIQNDKTNIMAEFKQLRDILDCEESNELQNLEKEEKNILKRLVQSENDMVLQTQSVRVLISDLERRL
QGSVVELLQDVDGVIKRIEKVTLQKPKTFLNEKRRVFRAPDLKRMLQVLKELTEVQRYWAHVTLVPSHPSYTIISEDGRQ
VRYQKPIRHLLVKVQYFYGVLGSPSITSGKHYWEVDVSNKRAWTLGVCVSLKCTANQSVSGTENYQPKNGYWVIGLRNAG
NYRAFQSSFEFRDFLAGSRLTLSPPLIVPLFMTICPNRVGVFLDYEARTISFFNVTSNGFLIYKFSDCHFSYPVFPYFNP
MTCELPMTLCSPRS
>Owl
MASRILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKKSMPHQGERSCPLCRISYSSENLRPNRHLVNIVERL
REVMLSPEEGQKVDHCAHHGEKLVLFCQQDGNVICWLCERSQEHRGHQTFLVEEVAQKYREKLQVALEMMRQKQKDAEKL
EADVREEQASWKIQIQNDKTNIMAEFKKRRDILDCEESKELQNLEKEEKNILKRLVQSENDMVLQTQSVRVLISDLEHRL
QGSVMELLQGVDGVIKRIEKVTLQNPKTFLNEKRRIFQTPDLKGTLQVFKEPTEVQRYWAHVTLVPSHPSCTVISEDERQ
VRYQKRIYQPFLKVKYFCGVLGSPSITSGKHYWEVDVSNKSEWILGVCVSLKRTASCSVPRIENDQPKNGYWVIGLRNAD
NYSAFQDAVEYSDFQDGSRSTPSAPLIVPLFMTICPNRVGVFLDYEACTVSFFNVTNNGFLIYKFSNCHFCYPVFPYFSP
MTCELPMTLCSPSS
>Titi
MASRILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESTLHQGERSCPLCRISYPSENLRPNRHLANIVERL
REVVLSPEEGQKVDLCARHGEKLLLFCQQDGNVICWLCERSQEHRGHHTFLVEEVAQTYRENLQVVLEMMRQKHQDAEKL
EADVREEQASWKIQIQNDKTNIMAEFKQLRDILDCEESNELQNLEKEEKNILKRLVQSENDMVLQTQSISVLISDLEHRL
QGSVMELLQGVDGVIKRVKNVTLQKPKTFLNEKRRVFRVPDLKGMLQVSKELTEVQRYWAHVTLVASHPSRAVISEDERQ
VRYQEWIHQSSGRVKYFYGVLGSPSITSGKHYWEVDVSNKSAWILGVCVSLKCAANRNGPGVENYQPKNGYWVIGLRNAD
NYSAFQDSVKYNDFQDGSRSTTYAPLIVPLFMTICPNRVGVFLDYEACTVSFFNVTSNGFLIYKFSNCHFSYPVFPYFSP
MTCELPMTLCSPRS
>Saki
MASRILMNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKKSMLHQGERSCPLCRISYPSENLRPNRHLANIVERL
REVMLSPEEGQKVDHCARHGEKLLLFCQQDGNVICWLCERSQEHRGHHTLLVEEVAQTYRENLQVALETMRQKQQDAEKL
EADVREEQASWKIQIRDDKTNIMAEFKQLRDILDCEESNELQILEKEEKNILKRLTQSENDMVLQTQSMGVLISDLEHRL
QGSVMELLQGVDEVIKRVKNVTLQKPKTFLNEKRRVFRAPDLKGMLQVFKELTEVQRYWVHVTLVPSHLSCAVISEDERQ
VRYQERIHQSFGKVKYFYGVLGSPSIRSGKHYWEVDVSNKSAWILGVCVSLKCTANRNGPRIENYQPKNGYWVIGLWNAG
NYSAFQDSVKYSDFQDGSHSATYGPLIVPLFMTICPNRVGVFLDYEACTVSFFNVTSNGFLIYKFSNCRFSDSVFPYFSP
MTCELPMTLCSPRS
>Howler
MASKILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESRERSCPLCRVSYHSENLRPNRHLANIAERLREVM
LSPEEGQKVDRCARHGEKLLLFCQQHGNVICWLCERSEEHRGHRTSLVEEVAQKYREKLQAALEMMRQKEQDAEMLEADV
REEQASWKIQIENDKTSTLAEFKQLRDILDCEESNELQKLEKEEENLLKRLVQSENDMVLQTQSIRVLIADLERRLQGSV
MELLQGVEGVIKRIKNVTLQKPETFLNEKRRVFQAPDLKGMLQVFKELKEVQCYWAHVTLIPNHPSCTVISEDKREVRYQ
EQIHHHPSMEVKYFYGILGSPSITSGKHYWEVDVSNKSAWILGVCVSLKCIGNFPGIENYQPQNGYWVIGLRNADNYSAF
QDAVPETENYQPKNRNRFTGLQNADNCSAFQNAFPGIQSYQPKKSHLFTGLQNLSNYNAFQNKVQYNYIDFQDDSLSTPS
APLIVPLFMTICPKRVGVFLDYEACTVSFFNVTSNGYLIYKFSNCQFSYPVFPYFSPMTCELPMTLCSPSS
>Spider
MASEILLNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESTLHQGERSCPLCRVSYQSENLRPNRHLANIAERL
REVMLSPEEGQKVDRCARHGEKLLLFCQQHGNVICWLCERSQEHRGHSTFLVEEVAQKYQEKLQVALEMMRQKQQDAEKL
EADVREEQASWKIQIENDKTNILAEFKQLRDILDCEESNELQNLEKEEENLLKTLAQSENDMVLQTQSMRVLIADLEHRL
QGSVMELLQDVEGVIKRIKNVTLQKPKTFLNEKRRVFRAPDLKGMLQVFKELKEVQCYWAHVTLVPSHPSCTVISEDERQ
VRYQEQIHQPSVKVKYFCGVLGSPGFTSGKHYWEVDVSDKSAWILGVCVSLKCTANVPGIENYQPKNGYWVIGLQNANNY
SAFQDAVPGTENYQPKNGNRRNKGLRNADNYSAFRDTFQPINDSWVTGLRNVDNYNAFQDAVKYSDFQDGSCSTPSAPLM
VPLFMTICPKRVGVFLDCKACTVSFFNVTSNGCLIYKFSKCHFSYPVFPYFSPMICKLPMTLCSPSS
>Woolly
MASEILVNIKEEVTCPICLDLLTEPLSLDCGHSFCQACITADHKESTLHQGERSCPLCRVGYQSENLRPNRHLANIAERL
REVMLSPEEGQKVDRCARHGEKLLLFCQQHGNVICWLCERSQEHRGHSTFLVEEVAQKYREKLQVALEMMREKQQDAEKL
EADVREEQASWKIQIKNDKTNILAEFKQLRDILDCEESNELQNLEKEEENLLKILAQSENDMVLQTQSMRVLIADLEHRL
QGSVMELLQGVEGIIKRTTNVTLQKPKTFLNEKRRVFRAPNLKGMLQVFKELKEVQCYWAHVTLVPSHPSCAVISEDQRQ
VRYQKQRHRPSVKAKYFYGVLGSPSFTSGKHYWEVDVSNKSAWILGVCVSLKCTANVPGIENYQPKNGYWVIGLQNADNY
SAFQDAVPGTEDYQPKNGCWRNTGLRNADNYSAFQDVFQPKNDYWVTGLWNADNYNAFQDAGKYSDFQDGSCSTPFAPLI
VPLFMTIRPKRVGVFLDYEACTVSFFNVTSNGCLIYKFSNCHFSCPVFPYFSPMTCKLPMTLCSPSS(base) suki@LAPTOP-5PMP3V9O:/mnt/c/Users/欧阳葭$ grepgrep ">" primatesAA.fasta | wc -l ## 22
grep: primatesAA.fasta: No such file or directory
0
(base) suki@LAPTOP-5PMP3V9O:/mnt/c/Users/欧阳葭$ ~/Dropbox/software/clustalw-2.1-macosx/clustalw2 -ALIGN -INFILE=primatesAA.fasta -OUTFILE=primatesAA-aligned.fasta -OUTPUT=FASTA
-bash: /home/suki/Dropbox/software/clustalw-2.1-macosx/clustalw2: No such file or directory
(base) suki@LAPTOP-5PMP3V9O:/mnt/c/Users/欧阳葭$ clustalw -ALIGN -INFILE=primatesAA.fasta -OUTFILE=primatesAA-aligned.fasta -OUTPUT=FASTA



 CLUSTAL 2.1 Multiple Sequence Alignments




ERROR: Cannot open input file. No alignment!


(base) suki@LAPTOP-5PMP3V9O:/mnt/c/Users/欧阳葭$ cd ~/phylogenetics-class/
(base) suki@LAPTOP-5PMP3V9O:~/phylogenetics-class$ cd ..
(base) suki@LAPTOP-5PMP3V9O:~$ cd /home/suki
(base) suki@LAPTOP-5PMP3V9O:~$ cd phylo-class
(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ ls
primatesAA.fasta  primatesAA.fasta:Zone.Identifier
(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ clustalw -ALIGN -INFILE=primatesAA.fasta -OUTFILE=primatesAA-aligned.fasta -OUTPUT=FASTA



 CLUSTAL 2.1 Multiple Sequence Alignments


Sequence format is Pearson
Sequence 1: Human        493 aa
Sequence 2: Chimp        493 aa
Sequence 3: Gorilla      493 aa
Sequence 4: Orangutan    493 aa
Sequence 5: Gibbon       494 aa
Sequence 6: Rhes_cDNA    497 aa
Sequence 7: Baboon       497 aa
Sequence 8: AGM          515 aa
Sequence 9: AGM_cDNA     515 aa
Sequence 10: Tant_cDNA    515 aa
Sequence 11: Patas        495 aa
Sequence 12: Colobus      495 aa
Sequence 13: DLangur      495 aa
Sequence 14: PMarmoset    494 aa
Sequence 15: Tamarin      494 aa
Sequence 16: Squirrel     494 aa
Sequence 17: Owl          494 aa
Sequence 18: Titi         494 aa
Sequence 19: Saki         494 aa
Sequence 20: Howler       551 aa
Sequence 21: Spider       547 aa
Sequence 22: Woolly       547 aa
Start of Pairwise alignments
Aligning...

Sequences (1:2) Aligned. Score:  97
Sequences (1:3) Aligned. Score:  97
Sequences (1:4) Aligned. Score:  93
Sequences (1:5) Aligned. Score:  90
Sequences (1:6) Aligned. Score:  87
Sequences (1:7) Aligned. Score:  87
Sequences (1:8) Aligned. Score:  87
Sequences (1:9) Aligned. Score:  87
Sequences (1:10) Aligned. Score:  87
Sequences (1:11) Aligned. Score:  86
Sequences (1:12) Aligned. Score:  88
Sequences (1:13) Aligned. Score:  88
Sequences (1:14) Aligned. Score:  71
Sequences (1:15) Aligned. Score:  70
Sequences (1:16) Aligned. Score:  68
Sequences (1:17) Aligned. Score:  70
Sequences (1:18) Aligned. Score:  70
Sequences (1:19) Aligned. Score:  71
Sequences (1:20) Aligned. Score:  68
Sequences (1:21) Aligned. Score:  69
Sequences (1:22) Aligned. Score:  69
Sequences (2:3) Aligned. Score:  98
Sequences (2:4) Aligned. Score:  94
Sequences (2:5) Aligned. Score:  90
Sequences (2:6) Aligned. Score:  87
Sequences (2:7) Aligned. Score:  87
Sequences (2:8) Aligned. Score:  86
Sequences (2:9) Aligned. Score:  87
Sequences (2:10) Aligned. Score:  87
Sequences (2:11) Aligned. Score:  87
Sequences (2:12) Aligned. Score:  88
Sequences (2:13) Aligned. Score:  88
Sequences (2:14) Aligned. Score:  70
Sequences (2:15) Aligned. Score:  69
Sequences (2:16) Aligned. Score:  67
Sequences (2:17) Aligned. Score:  70
Sequences (2:18) Aligned. Score:  70
Sequences (2:19) Aligned. Score:  70
Sequences (2:20) Aligned. Score:  69
Sequences (2:21) Aligned. Score:  69
Sequences (2:22) Aligned. Score:  69
Sequences (3:4) Aligned. Score:  94
Sequences (3:5) Aligned. Score:  91
Sequences (3:6) Aligned. Score:  87
Sequences (3:7) Aligned. Score:  88
Sequences (3:8) Aligned. Score:  87
Sequences (3:9) Aligned. Score:  87
Sequences (3:10) Aligned. Score:  87
Sequences (3:11) Aligned. Score:  87
Sequences (3:12) Aligned. Score:  89
Sequences (3:13) Aligned. Score:  88
Sequences (3:14) Aligned. Score:  71
Sequences (3:15) Aligned. Score:  70
Sequences (3:16) Aligned. Score:  68
Sequences (3:17) Aligned. Score:  71
Sequences (3:18) Aligned. Score:  71
Sequences (3:19) Aligned. Score:  72
Sequences (3:20) Aligned. Score:  69
Sequences (3:21) Aligned. Score:  70
Sequences (3:22) Aligned. Score:  70
Sequences (4:5) Aligned. Score:  90
Sequences (4:6) Aligned. Score:  86
Sequences (4:7) Aligned. Score:  86
Sequences (4:8) Aligned. Score:  86
Sequences (4:9) Aligned. Score:  86
Sequences (4:10) Aligned. Score:  86
Sequences (4:11) Aligned. Score:  86
Sequences (4:12) Aligned. Score:  87
Sequences (4:13) Aligned. Score:  87
Sequences (4:14) Aligned. Score:  70
Sequences (4:15) Aligned. Score:  69
Sequences (4:16) Aligned. Score:  67
Sequences (4:17) Aligned. Score:  68
Sequences (4:18) Aligned. Score:  69
Sequences (4:19) Aligned. Score:  69
Sequences (4:20) Aligned. Score:  69
Sequences (4:21) Aligned. Score:  69
Sequences (4:22) Aligned. Score:  69
Sequences (5:6) Aligned. Score:  83
Sequences (5:7) Aligned. Score:  84
Sequences (5:8) Aligned. Score:  83
Sequences (5:9) Aligned. Score:  83
Sequences (5:10) Aligned. Score:  83
Sequences (5:11) Aligned. Score:  83
Sequences (5:12) Aligned. Score:  85
Sequences (5:13) Aligned. Score:  85
Sequences (5:14) Aligned. Score:  69
Sequences (5:15) Aligned. Score:  68
Sequences (5:16) Aligned. Score:  67
Sequences (5:17) Aligned. Score:  68
Sequences (5:18) Aligned. Score:  70
Sequences (5:19) Aligned. Score:  69
Sequences (5:20) Aligned. Score:  67
Sequences (5:21) Aligned. Score:  69
Sequences (5:22) Aligned. Score:  67
Sequences (6:7) Aligned. Score:  97
Sequences (6:8) Aligned. Score:  96
Sequences (6:9) Aligned. Score:  96
Sequences (6:10) Aligned. Score:  96
Sequences (6:11) Aligned. Score:  94
Sequences (6:12) Aligned. Score:  95
Sequences (6:13) Aligned. Score:  95
Sequences (6:14) Aligned. Score:  69
Sequences (6:15) Aligned. Score:  68
Sequences (6:16) Aligned. Score:  66
Sequences (6:17) Aligned. Score:  68
Sequences (6:18) Aligned. Score:  70
Sequences (6:19) Aligned. Score:  70
Sequences (6:20) Aligned. Score:  65
Sequences (6:21) Aligned. Score:  68
Sequences (6:22) Aligned. Score:  68
Sequences (7:8) Aligned. Score:  96
Sequences (7:9) Aligned. Score:  96
Sequences (7:10) Aligned. Score:  96
Sequences (7:11) Aligned. Score:  95
Sequences (7:12) Aligned. Score:  95
Sequences (7:13) Aligned. Score:  95
Sequences (7:14) Aligned. Score:  70
Sequences (7:15) Aligned. Score:  68
Sequences (7:16) Aligned. Score:  67
Sequences (7:17) Aligned. Score:  68
Sequences (7:18) Aligned. Score:  70
Sequences (7:19) Aligned. Score:  71
Sequences (7:20) Aligned. Score:  66
Sequences (7:21) Aligned. Score:  69
Sequences (7:22) Aligned. Score:  69
Sequences (8:9) Aligned. Score:  98
Sequences (8:10) Aligned. Score:  99
Sequences (8:11) Aligned. Score:  94
Sequences (8:12) Aligned. Score:  94
Sequences (8:13) Aligned. Score:  94
Sequences (8:14) Aligned. Score:  70
Sequences (8:15) Aligned. Score:  69
Sequences (8:16) Aligned. Score:  67
Sequences (8:17) Aligned. Score:  68
Sequences (8:18) Aligned. Score:  70
Sequences (8:19) Aligned. Score:  70
Sequences (8:20) Aligned. Score:  65
Sequences (8:21) Aligned. Score:  66
Sequences (8:22) Aligned. Score:  67
Sequences (9:10) Aligned. Score:  98
Sequences (9:11) Aligned. Score:  94
Sequences (9:12) Aligned. Score:  95
Sequences (9:13) Aligned. Score:  95
Sequences (9:14) Aligned. Score:  70
Sequences (9:15) Aligned. Score:  69
Sequences (9:16) Aligned. Score:  67
Sequences (9:17) Aligned. Score:  68
Sequences (9:18) Aligned. Score:  71
Sequences (9:19) Aligned. Score:  71
Sequences (9:20) Aligned. Score:  65
Sequences (9:21) Aligned. Score:  66
Sequences (9:22) Aligned. Score:  67
Sequences (10:11) Aligned. Score:  94
Sequences (10:12) Aligned. Score:  94
Sequences (10:13) Aligned. Score:  94
Sequences (10:14) Aligned. Score:  70
Sequences (10:15) Aligned. Score:  69
Sequences (10:16) Aligned. Score:  67
Sequences (10:17) Aligned. Score:  69
Sequences (10:18) Aligned. Score:  70
Sequences (10:19) Aligned. Score:  71
Sequences (10:20) Aligned. Score:  65
Sequences (10:21) Aligned. Score:  67
Sequences (10:22) Aligned. Score:  67
Sequences (11:12) Aligned. Score:  94
Sequences (11:13) Aligned. Score:  94
Sequences (11:14) Aligned. Score:  69
Sequences (11:15) Aligned. Score:  67
Sequences (11:16) Aligned. Score:  66
Sequences (11:17) Aligned. Score:  68
Sequences (11:18) Aligned. Score:  69
Sequences (11:19) Aligned. Score:  69
Sequences (11:20) Aligned. Score:  66
Sequences (11:21) Aligned. Score:  68
Sequences (11:22) Aligned. Score:  68
Sequences (12:13) Aligned. Score:  98
Sequences (12:14) Aligned. Score:  71
Sequences (12:15) Aligned. Score:  69
Sequences (12:16) Aligned. Score:  67
Sequences (12:17) Aligned. Score:  70
Sequences (12:18) Aligned. Score:  71
Sequences (12:19) Aligned. Score:  72
Sequences (12:20) Aligned. Score:  67
Sequences (12:21) Aligned. Score:  70
Sequences (12:22) Aligned. Score:  70
Sequences (13:14) Aligned. Score:  70
Sequences (13:15) Aligned. Score:  69
Sequences (13:16) Aligned. Score:  67
Sequences (13:17) Aligned. Score:  70
Sequences (13:18) Aligned. Score:  71
Sequences (13:19) Aligned. Score:  71
Sequences (13:20) Aligned. Score:  67
Sequences (13:21) Aligned. Score:  70
Sequences (13:22) Aligned. Score:  70
Sequences (14:15) Aligned. Score:  93
Sequences (14:16) Aligned. Score:  86
Sequences (14:17) Aligned. Score:  88
Sequences (14:18) Aligned. Score:  89
Sequences (14:19) Aligned. Score:  87
Sequences (14:20) Aligned. Score:  83
Sequences (14:21) Aligned. Score:  86
Sequences (14:22) Aligned. Score:  85
Sequences (15:16) Aligned. Score:  84
Sequences (15:17) Aligned. Score:  87
Sequences (15:18) Aligned. Score:  87
Sequences (15:19) Aligned. Score:  87
Sequences (15:20) Aligned. Score:  82
Sequences (15:21) Aligned. Score:  86
Sequences (15:22) Aligned. Score:  85
Sequences (16:17) Aligned. Score:  85
Sequences (16:18) Aligned. Score:  85
Sequences (16:19) Aligned. Score:  84
Sequences (16:20) Aligned. Score:  80
Sequences (16:21) Aligned. Score:  83
Sequences (16:22) Aligned. Score:  82
Sequences (17:18) Aligned. Score:  88
Sequences (17:19) Aligned. Score:  87
Sequences (17:20) Aligned. Score:  84
Sequences (17:21) Aligned. Score:  86
Sequences (17:22) Aligned. Score:  85
Sequences (18:19) Aligned. Score:  92
Sequences (18:20) Aligned. Score:  85
Sequences (18:21) Aligned. Score:  87
Sequences (18:22) Aligned. Score:  87
Sequences (19:20) Aligned. Score:  83
Sequences (19:21) Aligned. Score:  86
Sequences (19:22) Aligned. Score:  85
Sequences (20:21) Aligned. Score:  86
Sequences (20:22) Aligned. Score:  85
Sequences (21:22) Aligned. Score:  92
Guide tree file created:   [primatesAA.dnd]

There are 21 groups
Start of Multiple Alignment

Aligning...
Group 1: Sequences:   2      Score:10695
Group 2: Sequences:   3      Score:10693
Group 3: Sequences:   4      Score:10550
Group 4: Sequences:   5      Score:10317
Group 5: Sequences:   2      Score:11293
Group 6: Sequences:   3      Score:11222
Group 7: Sequences:   2      Score:10814
Group 8: Sequences:   5      Score:10697
Group 9: Sequences:   6      Score:10568
Group 10: Sequences:   2      Score:10784
Group 11: Sequences:   8      Score:10576
Group 12: Sequences:  13      Score:9953
Group 13: Sequences:   2      Score:10473
Group 14: Sequences:   3      Score:10194
Group 15: Sequences:   4      Score:10034
Group 16: Sequences:   2      Score:10399
Group 17: Sequences:   6      Score:10117
Group 18: Sequences:   2      Score:11561
Group 19: Sequences:   3      Score:11013
Group 20: Sequences:   9      Score:9779
Group 21: Sequences:  22      Score:8717
Alignment Score 573733
firstres = 1 lastres = 587
FASTA file created!

Fasta-Alignment file created    [primatesAA-aligned.fasta]

(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ ls
primatesAA-aligned.fasta  primatesAA.dnd  primatesAA.fasta  primatesAA.fasta:Zone.Identifier
(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ cat primatesAA-aligned.fasta
>Human
MASGILVNVKEEVTCPICLELLTQPLSLDCGHSFCQACLTANHKKSMLDK-GESSCPVCR
ISYQPENIRPNRHVANIVEKLREVKLSPE-GQKVDHCARHGEKLLLFCQEDGKVICWLCE
RSQEHRGHHTFLTEEVAREYQVKLQAALEMLRQKQQEAEELEADIREEKASWKTQIQYDK
TNVLADFEQLRDILDWEESNELQNLEKEEEDILKSLTNSETEMVQQTQSLRELISDLEHR
LQGSVMELLQGVDGVIKRTENVTLKKPETFPKNQRRVFRAPDLKGMLEVFRELTDVRRYW
VDVTVAPNNISCAVISEDKRQVSSPKPQIIYGARGTRYQTFV------------------
--NFNYCTGILGSQSITSGKHYWEVDVSKKTAWILGVCAGFQPDAMCNIEKNENYQPKYG
YWVIGLEEGVKCSAFQDSS-----------------------------------------
-------------------------------FHTPSVPFIVPLSVIICPDRVGVFLDYEA
CTVSFFNITNHGFLIYKFSHCSFSQPVFPYLNPRKCGVPMTLCSPSS
>Chimp
MASGILVNVKEEVTCPICLELLTQPLSLDCGHSFCQACLTANHKKSMLDK-GESSCPVCR
ISYQPENIRPNRHVANIVEKLREVKLSPE-GQKVDHCAHHGEKLLLFCQEDGKVICWLCE
RSQEHRGHHTFLTEEVAREYQVKLQAALEMLRQKQQEAEELEADIREEKASWKTQIQYDK
TNVLADFEQLRDILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQSVRELISDLERR
LQGSVMELLQGVDGVIKRMENVTLKKPETFPKNQRRVFRAPDLKGMLEVFRELTDVRRYW
VDVTVAPNNISCAVISEDMRQVSSPKPQIIYGARGTRYQTFM------------------
--NFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMCNIEKNENYQPKYG
YWVIGLEEGVKCSAFQDGS-----------------------------------------
-------------------------------FHTPSAPFIVPLSVIICPDRVGVFLDYEA
CTVSFFNITNHGSLIYKFSHCSFSQPVFPYLNPRKCGVPMTLCSPSS
>Gorilla
MASGILVNVKEEVTCPICLELLTQPLSLDCGHSFCQACLTANHKKSMLDK-GESSCPVCR
ISYQPENIRPNRHVANIVEKLREVKLSPE-GQKVDHCARHGEKLLLFCQEDGKVICWLCE
RSQEHRGHHTFLTEEVAQEYQVKLQAALEMLRQKQQEAEELEADIREEKASWKTQIQYDK
TNVLADFEQLRDILDWEESNELQNLEKEEEDILKRLTKSETEMVQQTQSVRELISDLEHR
LQGSVMELLQGVDGVIKRMENVTLKKPETFPKNRRRVFRAPDLKGMLEVFRELTDVRRYW
VDVTVAPNNISCAVISEDMRQVSSPKPQIIYGAQGTRYQTFM------------------
--NFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDATCNIEKNENYQPKYG
YWVIGLEEGVKCSAFQDGS-----------------------------------------
-------------------------------FHTPSAPFIVPLSVIICPDRVGVFLDYEA
CTVSFFNITNHGFLIYKFSHCSFSQPVFPYLNPRKCRVPMTLCSPSS
>Orangutan
MASGILVNVKEEVTCPICLELLTQPLSLDCGHSFCQACLTANHKKSTLDK-GERSCPVCR
VSYQPKNIRPNRHVANIVEKLREVKLSPE-GQKVDHCARHGEKLLLFCKEDGKVICWLCE
RSQEHRGHHTFLTEEVAQKYQVKLQAALEMLRQKQQEAEELEADIREEKASWKTQIQYDK
TSVLADFEQLRDILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQSVRELISDVEHR
LQGSVMELLQGVDGIIKRMQNVTLKKPETFPKNQRRVFRAPNLKGMLEVFRELTDVRRYW
VDVTVAPNDISYAVISEDMRQVSCPEPQIIYGAQGTTYQTYV------------------
--NFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYNIEQNENYQPQYG
YWVIGLEEGVKCSAFQDGS-----------------------------------------
-------------------------------FHNPSAPFIVPLSVIICPDRVGVFLDYEA
CTVSFFNITNHGFLIYKFSHCSFSQPVFPYLNPRKCRVPMTLCSPSS
>Gibbon
MASGILVNVKEKVTCPICLELLTQPLSLDCGHSFCQACLTANHKTSMPDE-GERSCPVCR
ISYQHKNIRPNRHVANIVEKLREVKLSPEEGQKVDHCARHGKKLLLFCQEDRKVICWLCE
RSQEHRGHHTFLTEEVAQEYQMKLQAALQMLRQKQQEAEELEADIREEKASWKTQIQYDK
TNILADFEQLRHILDWVESNELQNLEKEEKDVLKRLMRSEIEMVQQTQSVRELISDLEHR
LQGSVMELLQGVDGVIKRMKNVTLKKPETFPKNRRRVFRAADLKVMLEVLRELRDVRRYW
VDVTVAPNNISYAVISEDMRQVSSPEPQIIFEAQGTISQTFV------------------
--NFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGLQPDAMYNIEQNENYQPKYG
YWVIGLEEGVKCNAFQDGS-----------------------------------------
-------------------------------IHTPSAPFVVPLSVNICPDRVGVFLDYEA
CTVSFFNITDHGFLIYKFSHCSFSQPVFPYLNPRKCTVPMTLCSPSS
>AGM
MASGILLNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHKESMLYKEEERSCPVCR
ISYQPENIQPNRHVANIVEKLREVKLSPEEGQKVDHCARHGEKLLLFCQEDSKVICWLCE
RSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEKLEADIREEKASWKIQIDYDK
TNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDVRRYW
VDVTLAPNNISHAVIAEDKRQVSYQNPQIMYQAPGSSFGSLTNFNYCTGVLGSQSITSRK
LTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDATYNIEQNENYQPKYG
YWVIGLQEGDKYSVFQDSS-----------------------------------------
-------------------------------SHTPFAPFIVPLSVIICPDRVGVFVDYEA
CTVSFFNITNHGFLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>Tant_cDNA
MASGILLNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHKESMLYKEEERSCPVCR
ISYQPENIQPNRHVANIVEKLREVKLSPEEGQKVDHCARHGEKLLLFCQEDSKVICWLCE
RSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEKLEADIREEKASWKIQIDYDK
TNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDVRRYW
VDVTLAPNNISHAVIAEDKRQVSYQNPQIMYQAPGSSFGSLTNFNYCTGVLGSQSITSRK
LTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDATYNIEQNENYQPKYG
YWVIGLQEGDKYSVFQDGS-----------------------------------------
-------------------------------SHTPFAPFIVPLSVIICPDRVGVFVDYEA
CTVSFFNITNHGFLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>AGM_cDNA
MASGILVNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHKESMLYKEEERSCPVCR
ISYQPENIQPNRHVANIVEKLREVKLSPEEGQKVDHCARHGEKLLLFCQEDSKVICWLCE
RSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEKLEADIREEKASWKIQIDYDK
TNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRVENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDVRRYW
VDVTLAPNNISHAVIAEDKRQVSYRNPQIMYQSPGSLFGSLTNFSYCTGVPGSQSITSGK
LTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDATYNIEQNENYQPKYG
YWVIGLQEGDKYSVFQDGS-----------------------------------------
-------------------------------SHTPFAPFIVPLSVIICPDRVGVFVDYEA
CTVSFFNITNHGFLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>Rhes_cDNA
MASGILLNVKEEVTCPICLELLTEPLSLHCGHSFCQACITANHKKSMLYKEGERSCPVCR
ISYQPENIQPNRHVANIVEKLREVKLSPEEGQKVDHCARHGEKLLLFCQEDSKVICWLCE
RSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEKLEADIREEKASWKIQIDYDK
TNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISELEHR
LQGSMMDLLQGVDGIIKRIENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDARRYW
VDVTLATNNISHAVIAEDKRQVSSRNPQIMYQAPGTLFTFPS------------------
LTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQSDAMYNIEQNENYQPKYG
YWVIGLQEGVKYSVFQDGS-----------------------------------------
-------------------------------SHTPFAPFIVPLSVIICPDRVGVFVDYEA
CTVSFFNITNHGFLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>Baboon
MASGILLNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHRKSMLYKEGERSCPVCR
ISYQPENIQPNRHVANIVEKLREVKLSPEEGLKVDHCARHGEKLLLFCQEDSKVICWLCE
RSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEKLEADIREEKASWKIQIDYDK
TNVSADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPKTFHKNQRRVFRAPDLKGMLDMFRELTDVRRYW
VDVTLAPNNISHAVIAEDKRQVSSRNPQITYQAPGTLFSFPS------------------
LTNFNYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYNIEQNENYQPKYG
YWVIGLQEGVKYSVFQDGS-----------------------------------------
-------------------------------SHTPFAPFIVPLSVIICPDRVGVFVDYEA
CTVSFFNITNHGFLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>Patas
MASGILLNVKEEVTCPICLELLTEPLSLPCGHSFCQACITANHKKSMLYKEEERSCPVCR
ISYQPENIQPNRHVANIVEKLREVKLSPEEGQKVDHCARHGEKLLLFCQEDRKVICWLCE
RSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEKLEADIREEKASWKIQIDYDK
TNVLADFEQLREILDWEESNELQYLEKEEEDILKSLTKSETKMVRQTQYVRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPETFHKNQRRVFRAPALKGMLDMFRELTDVRRYW
VDVTLAPNNISHVVIAEDKRQVSSRNPQIMYWAQGKLFQSLK------------------
--NFNYCTGILGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYDVEQNENYQPKYG
YWVIGLQEGVKYSVFQDGS-----------------------------------------
-------------------------------SHTPFAPFIAPLSVIFCPDRVGVFVDYEA
CTVSFFNITNHGFLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>Colobus
MASGILVNIKEEVTCPICLELLTEPLSLHCGHSFCQACITANHKKSMLYKEGERSCPVCR
ISYQPENIRPNRHVANIVEKLREVKLSPEEGQKVDHCARHGEKLLLFCQEDRKVICWLCE
RSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEKLEADIREEKASWKIQIDYDK
TNVLADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELVSDLEHR
LQGSVMELLQGVDGIIKRIEDMTLKKPKTFPKNQRRVFRAPDLKGMLDMFRELTDVRRYW
VDVTLAPNNISHAVIAEDKRRVSSPNPQIMYRAQGTLFQSLK------------------
--NFIYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYNIEQNENYQPKYG
YWVIGLQEGVKYSVFQDGS-----------------------------------------
-------------------------------SHTPFAPFIVPLSVIICPDRVGVFVDYEA
CTVSFFNITNHGFLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>DLangur
MASGILVNIKEEVTCPICLELLTEPLSLHCGHSFCQACITANHKKSMLYKEGERSCPVCR
ISYQPENIRPNRHVANIVEKLREVKLSPEEGQKVDHCARHGEKLLLFCQEDRKVICWLCE
RSQEHRGHHTFLMEEVAQEYHVKLQTALEMLRQKQQEAEKLEADIREEKASWKIQIDCDK
TNVLADFEQLREILDWEESNELQNLEKEEEDILKSLTKSETEMVQQTQYMRELISDLEHR
LQGSMMELLQGVDGIIKRIENMTLKKPKTFPKNQRRVFRAPDLKGILDMFRELTDVRRYW
VDVTLAPNNISHAVIAEDKRQVSSPNPQIMCRARGTLFQSLK------------------
--NFIYCTGVLGSQSITSGKHYWEVDVSKKSAWILGVCAGFQPDAMYNIEQNENYQPKYG
YWVIGLQEGVKYNVFQDGS-----------------------------------------
-------------------------------SHTPFAPFIVPLSVIICPDRVGVFVDYEA
CTVSFFNITNHGFLIYKFSQCSFSKPVFPYLNPRKCTVPMTLCSPSS
>PMarmoset
MASRILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESTLHQ-GERSCPLCR
MSYPSENLRPNRHLANIVERLKEVMLSPEEGQKVDHCARHGEKLLLFCQQDGNVICWLCE
RSQEHRGHHTFLVEEVAEKYQGKLQVALEMMRQKQQDAEKLEADVREEQASWKIQIQNDK
TNIMAEFKQLRDILDCEESKELQNLEKEEKNILKRLVQSESDMVLQTQSIRVLISDLERR
LQGSVMELLQGVDDVIKRIEKVTLQKPKTFLNEKRRVFRAPDLKGMLQAFKELTEVQRYW
AHVTLVPSHPSCTVISEDERQVRYQ-VPI-HQPLV-------------------------
--KVKYFYGVLGSLSITSGKHYWEVDVSNKRGWILGVCGSWKCNAKWNVLRPENYQPKNG
YWVIGLRNTDNYSAFQDAV-----------------------------------------
--------------------KYS--DVQDGSRSVSSGPLIVPLFMTICPNRVGVFLDYEA
CTISFFNVTSNGFLIYKFSNCHFSYPVFPYFSPTTCELPMTLCSPSS
>Tamarin
MASRILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESTPHQ-GERSCPLCR
MSYPSENLRPNRHLANIVERLKEVMLSPEEGQKVGHCARHGEKLLLFCEQDGNVICWLCE
RSQEHRGHHTLLVEEVAEKYQEKLQVALEMMRQKQQDAEKLEADVREEQASWKIQIRNDK
TNIMAEFKQLRDILDCEESKELQNLEKEEKNILKRLVQSESDMVLQTQSMRVLISDLERR
LQGSVLELLQGVDDVIKRIETVTLQKPKTFLNEKRRVFRAPDLKAMLQAFKELTEVQRYW
AHVTLVPSHPSYAVISEDERQVRYQ-FQI-HQPSV-------------------------
--KVNYFYGVLGSPSITSGKHYWEVDVTNKRDWILGICVSFKCNAKWNVLRPENYQPKNG
YWVIGLQNTNNYSAFQDAV-----------------------------------------
--------------------KYS--DFQIGSRSTASVPLIVPLFMTIYPNRVGVFLDYEA
CTVSFFNVTNNGFLIYKFSNCHFSYPVFPYFSPMTCELPMTLCSPSS
>Owl
MASRILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKKSMPHQ-GERSCPLCR
ISYSSENLRPNRHLVNIVERLREVMLSPEEGQKVDHCAHHGEKLVLFCQQDGNVICWLCE
RSQEHRGHQTFLVEEVAQKYREKLQVALEMMRQKQKDAEKLEADVREEQASWKIQIQNDK
TNIMAEFKKRRDILDCEESKELQNLEKEEKNILKRLVQSENDMVLQTQSVRVLISDLEHR
LQGSVMELLQGVDGVIKRIEKVTLQNPKTFLNEKRRIFQTPDLKGTLQVFKEPTEVQRYW
AHVTLVPSHPSCTVISEDERQVRYQ-KRI-YQPFL-------------------------
--KVKYFCGVLGSPSITSGKHYWEVDVSNKSEWILGVCVSLKRTASCSVPRIENDQPKNG
YWVIGLRNADNYSAFQDAV-----------------------------------------
--------------------EYS--DFQDGSRSTPSAPLIVPLFMTICPNRVGVFLDYEA
CTVSFFNVTNNGFLIYKFSNCHFCYPVFPYFSPMTCELPMTLCSPSS
>Squirrel
MASRILGSIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESMLHQ-GERSCPLCR
LPYQSENLRPNRHLASIVERLREVMLRPEERQNVDHCARHGEKLLLFCEQDGNIICWLCE
RSQEHRGHNTFLVEEVAQKYREKLQVALETMRQKQQDAEKLEADVRQEQASWKIQIQNDK
TNIMAEFKQLRDILDCEESNELQNLEKEEKNILKRLVQSENDMVLQTQSVRVLISDLERR
LQGSVVELLQDVDGVIKRIEKVTLQKPKTFLNEKRRVFRAPDLKRMLQVLKELTEVQRYW
AHVTLVPSHPSYTIISEDGRQVRYQ-KPI-RHLLV-------------------------
--KVQYFYGVLGSPSITSGKHYWEVDVSNKRAWTLGVCVSLKCTANQSVSGTENYQPKNG
YWVIGLRNAGNYRAFQSSF-----------------------------------------
--------------------EFR--DFLAGSRLTLSPPLIVPLFMTICPNRVGVFLDYEA
RTISFFNVTSNGFLIYKFSDCHFSYPVFPYFNPMTCELPMTLCSPRS
>Titi
MASRILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESTLHQ-GERSCPLCR
ISYPSENLRPNRHLANIVERLREVVLSPEEGQKVDLCARHGEKLLLFCQQDGNVICWLCE
RSQEHRGHHTFLVEEVAQTYRENLQVVLEMMRQKHQDAEKLEADVREEQASWKIQIQNDK
TNIMAEFKQLRDILDCEESNELQNLEKEEKNILKRLVQSENDMVLQTQSISVLISDLEHR
LQGSVMELLQGVDGVIKRVKNVTLQKPKTFLNEKRRVFRVPDLKGMLQVSKELTEVQRYW
AHVTLVASHPSRAVISEDERQVRYQ-EWI-HQSSG-------------------------
--RVKYFYGVLGSPSITSGKHYWEVDVSNKSAWILGVCVSLKCAANRNGPGVENYQPKNG
YWVIGLRNADNYSAFQDSV-----------------------------------------
--------------------KYN--DFQDGSRSTTYAPLIVPLFMTICPNRVGVFLDYEA
CTVSFFNVTSNGFLIYKFSNCHFSYPVFPYFSPMTCELPMTLCSPRS
>Saki
MASRILMNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKKSMLHQ-GERSCPLCR
ISYPSENLRPNRHLANIVERLREVMLSPEEGQKVDHCARHGEKLLLFCQQDGNVICWLCE
RSQEHRGHHTLLVEEVAQTYRENLQVALETMRQKQQDAEKLEADVREEQASWKIQIRDDK
TNIMAEFKQLRDILDCEESNELQILEKEEKNILKRLTQSENDMVLQTQSMGVLISDLEHR
LQGSVMELLQGVDEVIKRVKNVTLQKPKTFLNEKRRVFRAPDLKGMLQVFKELTEVQRYW
VHVTLVPSHLSCAVISEDERQVRYQ-ERI-HQSFG-------------------------
--KVKYFYGVLGSPSIRSGKHYWEVDVSNKSAWILGVCVSLKCTANRNGPRIENYQPKNG
YWVIGLWNAGNYSAFQDSV-----------------------------------------
--------------------KYS--DFQDGSHSATYGPLIVPLFMTICPNRVGVFLDYEA
CTVSFFNVTSNGFLIYKFSNCRFSDSVFPYFSPMTCELPMTLCSPRS
>Spider
MASEILLNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKESTLHQ-GERSCPLCR
VSYQSENLRPNRHLANIAERLREVMLSPEEGQKVDRCARHGEKLLLFCQQHGNVICWLCE
RSQEHRGHSTFLVEEVAQKYQEKLQVALEMMRQKQQDAEKLEADVREEQASWKIQIENDK
TNILAEFKQLRDILDCEESNELQNLEKEEENLLKTLAQSENDMVLQTQSMRVLIADLEHR
LQGSVMELLQDVEGVIKRIKNVTLQKPKTFLNEKRRVFRAPDLKGMLQVFKELKEVQCYW
AHVTLVPSHPSCTVISEDERQVRYQ-EQI-HQPSV-------------------------
--KVKYFCGVLGSPGFTSGKHYWEVDVSDKSAWILGVCVSLKCTAN--VPGIENYQPKNG
YWVIGLQNANNYSAFQDAVPGTENYQPKNGNRRNKGLRNADNYSAFRDTF------QPIN
DSWVTGLRNVDNYNAFQDAVKYS--DFQDGSCSTPSAPLMVPLFMTICPKRVGVFLDCKA
CTVSFFNVTSNGCLIYKFSKCHFSYPVFPYFSPMICKLPMTLCSPSS
>Woolly
MASEILVNIKEEVTCPICLDLLTEPLSLDCGHSFCQACITADHKESTLHQ-GERSCPLCR
VGYQSENLRPNRHLANIAERLREVMLSPEEGQKVDRCARHGEKLLLFCQQHGNVICWLCE
RSQEHRGHSTFLVEEVAQKYREKLQVALEMMREKQQDAEKLEADVREEQASWKIQIKNDK
TNILAEFKQLRDILDCEESNELQNLEKEEENLLKILAQSENDMVLQTQSMRVLIADLEHR
LQGSVMELLQGVEGIIKRTTNVTLQKPKTFLNEKRRVFRAPNLKGMLQVFKELKEVQCYW
AHVTLVPSHPSCAVISEDQRQVRYQ-KQR-HRPSV-------------------------
--KAKYFYGVLGSPSFTSGKHYWEVDVSNKSAWILGVCVSLKCTAN--VPGIENYQPKNG
YWVIGLQNADNYSAFQDAVPGTEDYQPKNGCWRNTGLRNADNYSAFQDVF------QPKN
DYWVTGLWNADNYNAFQDAGKYS--DFQDGSCSTPFAPLIVPLFMTIRPKRVGVFLDYEA
CTVSFFNVTSNGCLIYKFSNCHFSCPVFPYFSPMTCKLPMTLCSPSS
>Howler
MASKILVNIKEEVTCPICLELLTEPLSLDCGHSFCQACITANHKES-----RERSCPLCR
VSYHSENLRPNRHLANIAERLREVMLSPEEGQKVDRCARHGEKLLLFCQQHGNVICWLCE
RSEEHRGHRTSLVEEVAQKYREKLQAALEMMRQKEQDAEMLEADVREEQASWKIQIENDK
TSTLAEFKQLRDILDCEESNELQKLEKEEENLLKRLVQSENDMVLQTQSIRVLIADLERR
LQGSVMELLQGVEGVIKRIKNVTLQKPETFLNEKRRVFQAPDLKGMLQVFKELKEVQCYW
AHVTLIPNHPSCTVISEDKREVRYQ-EQIHHHPSM-------------------------
--EVKYFYGILGSPSITSGKHYWEVDVSNKSAWILGVCVSLKCIGN--FPGIENYQPQNG
YWVIGLRNADNYSAFQDAVPETENYQPKNRN-RFTGLQNADNCSAFQNAFPGIQSYQPKK
SHLFTGLQNLSNYNAFQNKVQYNYIDFQDDSLSTPSAPLIVPLFMTICPKRVGVFLDYEA
CTVSFFNVTSNGYLIYKFSNCQFSYPVFPYFSPMTCELPMTLCSPSS
(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ clustalw -ALIGN -INFILE=primatesAA.fasta -OUTFILE=primatesAA-aligned.aln -OUTPUT=CLUSTAL



 CLUSTAL 2.1 Multiple Sequence Alignments


Sequence format is Pearson
Sequence 1: Human        493 aa
Sequence 2: Chimp        493 aa
Sequence 3: Gorilla      493 aa
Sequence 4: Orangutan    493 aa
Sequence 5: Gibbon       494 aa
Sequence 6: Rhes_cDNA    497 aa
Sequence 7: Baboon       497 aa
Sequence 8: AGM          515 aa
Sequence 9: AGM_cDNA     515 aa
Sequence 10: Tant_cDNA    515 aa
Sequence 11: Patas        495 aa
Sequence 12: Colobus      495 aa
Sequence 13: DLangur      495 aa
Sequence 14: PMarmoset    494 aa
Sequence 15: Tamarin      494 aa
Sequence 16: Squirrel     494 aa
Sequence 17: Owl          494 aa
Sequence 18: Titi         494 aa
Sequence 19: Saki         494 aa
Sequence 20: Howler       551 aa
Sequence 21: Spider       547 aa
Sequence 22: Woolly       547 aa
Start of Pairwise alignments
Aligning...

Sequences (1:2) Aligned. Score:  97
Sequences (1:3) Aligned. Score:  97
Sequences (1:4) Aligned. Score:  93
Sequences (1:5) Aligned. Score:  90
Sequences (1:6) Aligned. Score:  87
Sequences (1:7) Aligned. Score:  87
Sequences (1:8) Aligned. Score:  87
Sequences (1:9) Aligned. Score:  87
Sequences (1:10) Aligned. Score:  87
Sequences (1:11) Aligned. Score:  86
Sequences (1:12) Aligned. Score:  88
Sequences (1:13) Aligned. Score:  88
Sequences (1:14) Aligned. Score:  71
Sequences (1:15) Aligned. Score:  70
Sequences (1:16) Aligned. Score:  68
Sequences (1:17) Aligned. Score:  70
Sequences (1:18) Aligned. Score:  70
Sequences (1:19) Aligned. Score:  71
Sequences (1:20) Aligned. Score:  68
Sequences (1:21) Aligned. Score:  69
Sequences (1:22) Aligned. Score:  69
Sequences (2:3) Aligned. Score:  98
Sequences (2:4) Aligned. Score:  94
Sequences (2:5) Aligned. Score:  90
Sequences (2:6) Aligned. Score:  87
Sequences (2:7) Aligned. Score:  87
Sequences (2:8) Aligned. Score:  86
Sequences (2:9) Aligned. Score:  87
Sequences (2:10) Aligned. Score:  87
Sequences (2:11) Aligned. Score:  87
Sequences (2:12) Aligned. Score:  88
Sequences (2:13) Aligned. Score:  88
Sequences (2:14) Aligned. Score:  70
Sequences (2:15) Aligned. Score:  69
Sequences (2:16) Aligned. Score:  67
Sequences (2:17) Aligned. Score:  70
Sequences (2:18) Aligned. Score:  70
Sequences (2:19) Aligned. Score:  70
Sequences (2:20) Aligned. Score:  69
Sequences (2:21) Aligned. Score:  69
Sequences (2:22) Aligned. Score:  69
Sequences (3:4) Aligned. Score:  94
Sequences (3:5) Aligned. Score:  91
Sequences (3:6) Aligned. Score:  87
Sequences (3:7) Aligned. Score:  88
Sequences (3:8) Aligned. Score:  87
Sequences (3:9) Aligned. Score:  87
Sequences (3:10) Aligned. Score:  87
Sequences (3:11) Aligned. Score:  87
Sequences (3:12) Aligned. Score:  89
Sequences (3:13) Aligned. Score:  88
Sequences (3:14) Aligned. Score:  71
Sequences (3:15) Aligned. Score:  70
Sequences (3:16) Aligned. Score:  68
Sequences (3:17) Aligned. Score:  71
Sequences (3:18) Aligned. Score:  71
Sequences (3:19) Aligned. Score:  72
Sequences (3:20) Aligned. Score:  69
Sequences (3:21) Aligned. Score:  70
Sequences (3:22) Aligned. Score:  70
Sequences (4:5) Aligned. Score:  90
Sequences (4:6) Aligned. Score:  86
Sequences (4:7) Aligned. Score:  86
Sequences (4:8) Aligned. Score:  86
Sequences (4:9) Aligned. Score:  86
Sequences (4:10) Aligned. Score:  86
Sequences (4:11) Aligned. Score:  86
Sequences (4:12) Aligned. Score:  87
Sequences (4:13) Aligned. Score:  87
Sequences (4:14) Aligned. Score:  70
Sequences (4:15) Aligned. Score:  69
Sequences (4:16) Aligned. Score:  67
Sequences (4:17) Aligned. Score:  68
Sequences (4:18) Aligned. Score:  69
Sequences (4:19) Aligned. Score:  69
Sequences (4:20) Aligned. Score:  69
Sequences (4:21) Aligned. Score:  69
Sequences (4:22) Aligned. Score:  69
Sequences (5:6) Aligned. Score:  83
Sequences (5:7) Aligned. Score:  84
Sequences (5:8) Aligned. Score:  83
Sequences (5:9) Aligned. Score:  83
Sequences (5:10) Aligned. Score:  83
Sequences (5:11) Aligned. Score:  83
Sequences (5:12) Aligned. Score:  85
Sequences (5:13) Aligned. Score:  85
Sequences (5:14) Aligned. Score:  69
Sequences (5:15) Aligned. Score:  68
Sequences (5:16) Aligned. Score:  67
Sequences (5:17) Aligned. Score:  68
Sequences (5:18) Aligned. Score:  70
Sequences (5:19) Aligned. Score:  69
Sequences (5:20) Aligned. Score:  67
Sequences (5:21) Aligned. Score:  69
Sequences (5:22) Aligned. Score:  67
Sequences (6:7) Aligned. Score:  97
Sequences (6:8) Aligned. Score:  96
Sequences (6:9) Aligned. Score:  96
Sequences (6:10) Aligned. Score:  96
Sequences (6:11) Aligned. Score:  94
Sequences (6:12) Aligned. Score:  95
Sequences (6:13) Aligned. Score:  95
Sequences (6:14) Aligned. Score:  69
Sequences (6:15) Aligned. Score:  68
Sequences (6:16) Aligned. Score:  66
Sequences (6:17) Aligned. Score:  68
Sequences (6:18) Aligned. Score:  70
Sequences (6:19) Aligned. Score:  70
Sequences (6:20) Aligned. Score:  65
Sequences (6:21) Aligned. Score:  68
Sequences (6:22) Aligned. Score:  68
Sequences (7:8) Aligned. Score:  96
Sequences (7:9) Aligned. Score:  96
Sequences (7:10) Aligned. Score:  96
Sequences (7:11) Aligned. Score:  95
Sequences (7:12) Aligned. Score:  95
Sequences (7:13) Aligned. Score:  95
Sequences (7:14) Aligned. Score:  70
Sequences (7:15) Aligned. Score:  68
Sequences (7:16) Aligned. Score:  67
Sequences (7:17) Aligned. Score:  68
Sequences (7:18) Aligned. Score:  70
Sequences (7:19) Aligned. Score:  71
Sequences (7:20) Aligned. Score:  66
Sequences (7:21) Aligned. Score:  69
Sequences (7:22) Aligned. Score:  69
Sequences (8:9) Aligned. Score:  98
Sequences (8:10) Aligned. Score:  99
Sequences (8:11) Aligned. Score:  94
Sequences (8:12) Aligned. Score:  94
Sequences (8:13) Aligned. Score:  94
Sequences (8:14) Aligned. Score:  70
Sequences (8:15) Aligned. Score:  69
Sequences (8:16) Aligned. Score:  67
Sequences (8:17) Aligned. Score:  68
Sequences (8:18) Aligned. Score:  70
Sequences (8:19) Aligned. Score:  70
Sequences (8:20) Aligned. Score:  65
Sequences (8:21) Aligned. Score:  66
Sequences (8:22) Aligned. Score:  67
Sequences (9:10) Aligned. Score:  98
Sequences (9:11) Aligned. Score:  94
Sequences (9:12) Aligned. Score:  95
Sequences (9:13) Aligned. Score:  95
Sequences (9:14) Aligned. Score:  70
Sequences (9:15) Aligned. Score:  69
Sequences (9:16) Aligned. Score:  67
Sequences (9:17) Aligned. Score:  68
Sequences (9:18) Aligned. Score:  71
Sequences (9:19) Aligned. Score:  71
Sequences (9:20) Aligned. Score:  65
Sequences (9:21) Aligned. Score:  66
Sequences (9:22) Aligned. Score:  67
Sequences (10:11) Aligned. Score:  94
Sequences (10:12) Aligned. Score:  94
Sequences (10:13) Aligned. Score:  94
Sequences (10:14) Aligned. Score:  70
Sequences (10:15) Aligned. Score:  69
Sequences (10:16) Aligned. Score:  67
Sequences (10:17) Aligned. Score:  69
Sequences (10:18) Aligned. Score:  70
Sequences (10:19) Aligned. Score:  71
Sequences (10:20) Aligned. Score:  65
Sequences (10:21) Aligned. Score:  67
Sequences (10:22) Aligned. Score:  67
Sequences (11:12) Aligned. Score:  94
Sequences (11:13) Aligned. Score:  94
Sequences (11:14) Aligned. Score:  69
Sequences (11:15) Aligned. Score:  67
Sequences (11:16) Aligned. Score:  66
Sequences (11:17) Aligned. Score:  68
Sequences (11:18) Aligned. Score:  69
Sequences (11:19) Aligned. Score:  69
Sequences (11:20) Aligned. Score:  66
Sequences (11:21) Aligned. Score:  68
Sequences (11:22) Aligned. Score:  68
Sequences (12:13) Aligned. Score:  98
Sequences (12:14) Aligned. Score:  71
Sequences (12:15) Aligned. Score:  69
Sequences (12:16) Aligned. Score:  67
Sequences (12:17) Aligned. Score:  70
Sequences (12:18) Aligned. Score:  71
Sequences (12:19) Aligned. Score:  72
Sequences (12:20) Aligned. Score:  67
Sequences (12:21) Aligned. Score:  70
Sequences (12:22) Aligned. Score:  70
Sequences (13:14) Aligned. Score:  70
Sequences (13:15) Aligned. Score:  69
Sequences (13:16) Aligned. Score:  67
Sequences (13:17) Aligned. Score:  70
Sequences (13:18) Aligned. Score:  71
Sequences (13:19) Aligned. Score:  71
Sequences (13:20) Aligned. Score:  67
Sequences (13:21) Aligned. Score:  70
Sequences (13:22) Aligned. Score:  70
Sequences (14:15) Aligned. Score:  93
Sequences (14:16) Aligned. Score:  86
Sequences (14:17) Aligned. Score:  88
Sequences (14:18) Aligned. Score:  89
Sequences (14:19) Aligned. Score:  87
Sequences (14:20) Aligned. Score:  83
Sequences (14:21) Aligned. Score:  86
Sequences (14:22) Aligned. Score:  85
Sequences (15:16) Aligned. Score:  84
Sequences (15:17) Aligned. Score:  87
Sequences (15:18) Aligned. Score:  87
Sequences (15:19) Aligned. Score:  87
Sequences (15:20) Aligned. Score:  82
Sequences (15:21) Aligned. Score:  86
Sequences (15:22) Aligned. Score:  85
Sequences (16:17) Aligned. Score:  85
Sequences (16:18) Aligned. Score:  85
Sequences (16:19) Aligned. Score:  84
Sequences (16:20) Aligned. Score:  80
Sequences (16:21) Aligned. Score:  83
Sequences (16:22) Aligned. Score:  82
Sequences (17:18) Aligned. Score:  88
Sequences (17:19) Aligned. Score:  87
Sequences (17:20) Aligned. Score:  84
Sequences (17:21) Aligned. Score:  86
Sequences (17:22) Aligned. Score:  85
Sequences (18:19) Aligned. Score:  92
Sequences (18:20) Aligned. Score:  85
Sequences (18:21) Aligned. Score:  87
Sequences (18:22) Aligned. Score:  87
Sequences (19:20) Aligned. Score:  83
Sequences (19:21) Aligned. Score:  86
Sequences (19:22) Aligned. Score:  85
Sequences (20:21) Aligned. Score:  86
Sequences (20:22) Aligned. Score:  85
Sequences (21:22) Aligned. Score:  92
Guide tree file created:   [primatesAA.dnd]

There are 21 groups
Start of Multiple Alignment

Aligning...
Group 1: Sequences:   2      Score:10695
Group 2: Sequences:   3      Score:10693
Group 3: Sequences:   4      Score:10550
Group 4: Sequences:   5      Score:10317
Group 5: Sequences:   2      Score:11293
Group 6: Sequences:   3      Score:11222
Group 7: Sequences:   2      Score:10814
Group 8: Sequences:   5      Score:10697
Group 9: Sequences:   6      Score:10568
Group 10: Sequences:   2      Score:10784
Group 11: Sequences:   8      Score:10576
Group 12: Sequences:  13      Score:9953
Group 13: Sequences:   2      Score:10473
Group 14: Sequences:   3      Score:10194
Group 15: Sequences:   4      Score:10034
Group 16: Sequences:   2      Score:10399
Group 17: Sequences:   6      Score:10117
Group 18: Sequences:   2      Score:11561
Group 19: Sequences:   3      Score:11013
Group 20: Sequences:   9      Score:9779
Group 21: Sequences:  22      Score:8717
Alignment Score 573733

CLUSTAL-Alignment file created  [primatesAA-aligned.aln]

(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ ls
primatesAA-aligned.aln  primatesAA-aligned.fasta  primatesAA.dnd  primatesAA.fasta  primatesAA.fasta:Zone.Identifier
(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ clustalw -TREE -INFILE=primatesAA.fasta -OUTPUTTREE=nj



 CLUSTAL 2.1 Multiple Sequence Alignments


Sequence format is Pearson
Sequence 1: Human        493 aa
Sequence 2: Chimp        493 aa
Sequence 3: Gorilla      493 aa
Sequence 4: Orangutan    493 aa
Sequence 5: Gibbon       494 aa
Sequence 6: Rhes_cDNA    497 aa
Sequence 7: Baboon       497 aa
Sequence 8: AGM          515 aa
Sequence 9: AGM_cDNA     515 aa
Sequence 10: Tant_cDNA    515 aa
Sequence 11: Patas        495 aa
Sequence 12: Colobus      495 aa
Sequence 13: DLangur      495 aa
Sequence 14: PMarmoset    494 aa
Sequence 15: Tamarin      494 aa
Sequence 16: Squirrel     494 aa
Sequence 17: Owl          494 aa
Sequence 18: Titi         494 aa
Sequence 19: Saki         494 aa
Sequence 20: Howler       551 aa
Sequence 21: Spider       547 aa
Sequence 22: Woolly       547 aa

Phylogenetic tree file created:   [primatesAA.nj]

Phylogenetic tree file created:   [primatesAA.ph]

(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ ls
primatesAA-aligned.aln    primatesAA.dnd    primatesAA.fasta:Zone.Identifier  primatesAA.ph
primatesAA-aligned.fasta  primatesAA.fasta  primatesAA.nj
(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ cat primatesAA.nj


 DIST   = percentage divergence (/100)
 Length = number of sites used in comparison

   1 vs.   2  DIST = 0.0223;  length =    493
   1 vs.   3  DIST = 0.0284;  length =    493
   1 vs.   4  DIST = 0.0629;  length =    493
   1 vs.   5  DIST = 0.7850;  length =    493
   1 vs.   6  DIST = 0.8641;  length =    493
   1 vs.   7  DIST = 0.8682;  length =    493
   1 vs.   8  DIST = 0.8621;  length =    493
   1 vs.   9  DIST = 0.8600;  length =    493
   1 vs.  10  DIST = 0.8621;  length =    493
   1 vs.  11  DIST = 0.8661;  length =    493
   1 vs.  12  DIST = 0.8641;  length =    493
   1 vs.  13  DIST = 0.8641;  length =    493
   1 vs.  14  DIST = 0.8134;  length =    493
   1 vs.  15  DIST = 0.8134;  length =    493
   1 vs.  16  DIST = 0.8195;  length =    493
   1 vs.  17  DIST = 0.8134;  length =    493
   1 vs.  18  DIST = 0.8215;  length =    493
   1 vs.  19  DIST = 0.8114;  length =    493
   1 vs.  20  DIST = 0.8499;  length =    493
   1 vs.  21  DIST = 0.8073;  length =    493
   1 vs.  22  DIST = 0.8093;  length =    493
   2 vs.   3  DIST = 0.0183;  length =    493
   2 vs.   4  DIST = 0.0568;  length =    493
   2 vs.   5  DIST = 0.7850;  length =    493
   2 vs.   6  DIST = 0.8641;  length =    493
   2 vs.   7  DIST = 0.8682;  length =    493
   2 vs.   8  DIST = 0.8641;  length =    493
   2 vs.   9  DIST = 0.8621;  length =    493
   2 vs.  10  DIST = 0.8641;  length =    493
   2 vs.  11  DIST = 0.8661;  length =    493
   2 vs.  12  DIST = 0.8641;  length =    493
   2 vs.  13  DIST = 0.8641;  length =    493
   2 vs.  14  DIST = 0.8154;  length =    493
   2 vs.  15  DIST = 0.8154;  length =    493
   2 vs.  16  DIST = 0.8215;  length =    493
   2 vs.  17  DIST = 0.8154;  length =    493
   2 vs.  18  DIST = 0.8235;  length =    493
   2 vs.  19  DIST = 0.8134;  length =    493
   2 vs.  20  DIST = 0.8540;  length =    493
   2 vs.  21  DIST = 0.8093;  length =    493
   2 vs.  22  DIST = 0.8114;  length =    493
   3 vs.   4  DIST = 0.0507;  length =    493
   3 vs.   5  DIST = 0.7850;  length =    493
   3 vs.   6  DIST = 0.8621;  length =    493
   3 vs.   7  DIST = 0.8661;  length =    493
   3 vs.   8  DIST = 0.8621;  length =    493
   3 vs.   9  DIST = 0.8600;  length =    493
   3 vs.  10  DIST = 0.8621;  length =    493
   3 vs.  11  DIST = 0.8661;  length =    493
   3 vs.  12  DIST = 0.8661;  length =    493
   3 vs.  13  DIST = 0.8661;  length =    493
   3 vs.  14  DIST = 0.8154;  length =    493
   3 vs.  15  DIST = 0.8154;  length =    493
   3 vs.  16  DIST = 0.8215;  length =    493
   3 vs.  17  DIST = 0.8154;  length =    493
   3 vs.  18  DIST = 0.8235;  length =    493
   3 vs.  19  DIST = 0.8134;  length =    493
   3 vs.  20  DIST = 0.8560;  length =    493
   3 vs.  21  DIST = 0.8093;  length =    493
   3 vs.  22  DIST = 0.8114;  length =    493
   4 vs.   5  DIST = 0.7870;  length =    493
   4 vs.   6  DIST = 0.8641;  length =    493
   4 vs.   7  DIST = 0.8682;  length =    493
   4 vs.   8  DIST = 0.8661;  length =    493
   4 vs.   9  DIST = 0.8641;  length =    493
   4 vs.  10  DIST = 0.8661;  length =    493
   4 vs.  11  DIST = 0.8661;  length =    493
   4 vs.  12  DIST = 0.8661;  length =    493
   4 vs.  13  DIST = 0.8661;  length =    493
   4 vs.  14  DIST = 0.8114;  length =    493
   4 vs.  15  DIST = 0.8114;  length =    493
   4 vs.  16  DIST = 0.8195;  length =    493
   4 vs.  17  DIST = 0.8174;  length =    493
   4 vs.  18  DIST = 0.8195;  length =    493
   4 vs.  19  DIST = 0.8134;  length =    493
   4 vs.  20  DIST = 0.8580;  length =    493
   4 vs.  21  DIST = 0.8032;  length =    493
   4 vs.  22  DIST = 0.8053;  length =    493
   5 vs.   6  DIST = 0.8540;  length =    493
   5 vs.   7  DIST = 0.8580;  length =    493
   5 vs.   8  DIST = 0.8621;  length =    493
   5 vs.   9  DIST = 0.8621;  length =    493
   5 vs.  10  DIST = 0.8621;  length =    493
   5 vs.  11  DIST = 0.8661;  length =    493
   5 vs.  12  DIST = 0.8702;  length =    493
   5 vs.  13  DIST = 0.8682;  length =    493
   5 vs.  14  DIST = 0.3915;  length =    493
   5 vs.  15  DIST = 0.3955;  length =    493
   5 vs.  16  DIST = 0.4138;  length =    493
   5 vs.  17  DIST = 0.3996;  length =    493
   5 vs.  18  DIST = 0.3834;  length =    493
   5 vs.  19  DIST = 0.3915;  length =    493
   5 vs.  20  DIST = 0.8519;  length =    493
   5 vs.  21  DIST = 0.5010;  length =    493
   5 vs.  22  DIST = 0.5152;  length =    493
   6 vs.   7  DIST = 0.0203;  length =    493
   6 vs.   8  DIST = 0.3103;  length =    493
   6 vs.   9  DIST = 0.3144;  length =    493
   6 vs.  10  DIST = 0.3103;  length =    493
   6 vs.  11  DIST = 0.3347;  length =    493
   6 vs.  12  DIST = 0.3347;  length =    493
   6 vs.  13  DIST = 0.3327;  length =    493
   6 vs.  14  DIST = 0.8600;  length =    493
   6 vs.  15  DIST = 0.8641;  length =    493
   6 vs.  16  DIST = 0.8641;  length =    493
   6 vs.  17  DIST = 0.8600;  length =    493
   6 vs.  18  DIST = 0.8580;  length =    493
   6 vs.  19  DIST = 0.8580;  length =    493
   6 vs.  20  DIST = 0.8641;  length =    493
   6 vs.  21  DIST = 0.8580;  length =    493
   6 vs.  22  DIST = 0.8621;  length =    493
   7 vs.   8  DIST = 0.3063;  length =    493
   7 vs.   9  DIST = 0.3103;  length =    493
   7 vs.  10  DIST = 0.3063;  length =    493
   7 vs.  11  DIST = 0.3306;  length =    493
   7 vs.  12  DIST = 0.3347;  length =    493
   7 vs.  13  DIST = 0.3327;  length =    493
   7 vs.  14  DIST = 0.8621;  length =    493
   7 vs.  15  DIST = 0.8661;  length =    493
   7 vs.  16  DIST = 0.8661;  length =    493
   7 vs.  17  DIST = 0.8621;  length =    493
   7 vs.  18  DIST = 0.8600;  length =    493
   7 vs.  19  DIST = 0.8600;  length =    493
   7 vs.  20  DIST = 0.8682;  length =    493
   7 vs.  21  DIST = 0.8580;  length =    493
   7 vs.  22  DIST = 0.8621;  length =    493
   8 vs.   9  DIST = 0.0183;  length =    493
   8 vs.  10  DIST = 0.0020;  length =    493
   8 vs.  11  DIST = 0.2961;  length =    493
   8 vs.  12  DIST = 0.3043;  length =    493
   8 vs.  13  DIST = 0.3022;  length =    493
   8 vs.  14  DIST = 0.8560;  length =    493
   8 vs.  15  DIST = 0.8641;  length =    493
   8 vs.  16  DIST = 0.8540;  length =    493
   8 vs.  17  DIST = 0.8600;  length =    493
   8 vs.  18  DIST = 0.8479;  length =    493
   8 vs.  19  DIST = 0.8499;  length =    493
   8 vs.  20  DIST = 0.8621;  length =    493
   8 vs.  21  DIST = 0.8560;  length =    493
   8 vs.  22  DIST = 0.8661;  length =    493
   9 vs.  10  DIST = 0.0162;  length =    493
   9 vs.  11  DIST = 0.3002;  length =    493
   9 vs.  12  DIST = 0.3043;  length =    493
   9 vs.  13  DIST = 0.3022;  length =    493
   9 vs.  14  DIST = 0.8519;  length =    493
   9 vs.  15  DIST = 0.8600;  length =    493
   9 vs.  16  DIST = 0.8519;  length =    493
   9 vs.  17  DIST = 0.8560;  length =    493
   9 vs.  18  DIST = 0.8438;  length =    493
   9 vs.  19  DIST = 0.8479;  length =    493
   9 vs.  20  DIST = 0.8580;  length =    493
   9 vs.  21  DIST = 0.8580;  length =    493
   9 vs.  22  DIST = 0.8621;  length =    493
  10 vs.  11  DIST = 0.2961;  length =    493
  10 vs.  12  DIST = 0.3043;  length =    493
  10 vs.  13  DIST = 0.3022;  length =    493
  10 vs.  14  DIST = 0.8560;  length =    493
  10 vs.  15  DIST = 0.8641;  length =    493
  10 vs.  16  DIST = 0.8540;  length =    493
  10 vs.  17  DIST = 0.8600;  length =    493
  10 vs.  18  DIST = 0.8479;  length =    493
  10 vs.  19  DIST = 0.8499;  length =    493
  10 vs.  20  DIST = 0.8621;  length =    493
  10 vs.  21  DIST = 0.8560;  length =    493
  10 vs.  22  DIST = 0.8661;  length =    493
  11 vs.  12  DIST = 0.0527;  length =    493
  11 vs.  13  DIST = 0.0548;  length =    493
  11 vs.  14  DIST = 0.8519;  length =    493
  11 vs.  15  DIST = 0.8621;  length =    493
  11 vs.  16  DIST = 0.8621;  length =    493
  11 vs.  17  DIST = 0.8600;  length =    493
  11 vs.  18  DIST = 0.8499;  length =    493
  11 vs.  19  DIST = 0.8458;  length =    493
  11 vs.  20  DIST = 0.8682;  length =    493
  11 vs.  21  DIST = 0.8499;  length =    493
  11 vs.  22  DIST = 0.8540;  length =    493
  12 vs.  13  DIST = 0.0183;  length =    493
  12 vs.  14  DIST = 0.8519;  length =    493
  12 vs.  15  DIST = 0.8600;  length =    493
  12 vs.  16  DIST = 0.8580;  length =    493
  12 vs.  17  DIST = 0.8600;  length =    493
  12 vs.  18  DIST = 0.8499;  length =    493
  12 vs.  19  DIST = 0.8479;  length =    493
  12 vs.  20  DIST = 0.8661;  length =    493
  12 vs.  21  DIST = 0.8540;  length =    493
  12 vs.  22  DIST = 0.8540;  length =    493
  13 vs.  14  DIST = 0.8499;  length =    493
  13 vs.  15  DIST = 0.8600;  length =    493
  13 vs.  16  DIST = 0.8600;  length =    493
  13 vs.  17  DIST = 0.8580;  length =    493
  13 vs.  18  DIST = 0.8479;  length =    493
  13 vs.  19  DIST = 0.8458;  length =    493
  13 vs.  20  DIST = 0.8661;  length =    493
  13 vs.  21  DIST = 0.8519;  length =    493
  13 vs.  22  DIST = 0.8519;  length =    493
  14 vs.  15  DIST = 0.0669;  length =    493
  14 vs.  16  DIST = 0.1400;  length =    493
  14 vs.  17  DIST = 0.1136;  length =    493
  14 vs.  18  DIST = 0.1095;  length =    493
  14 vs.  19  DIST = 0.1237;  length =    493
  14 vs.  20  DIST = 0.8479;  length =    493
  14 vs.  21  DIST = 0.3185;  length =    493
  14 vs.  22  DIST = 0.3367;  length =    493
  15 vs.  16  DIST = 0.1521;  length =    493
  15 vs.  17  DIST = 0.1278;  length =    493
  15 vs.  18  DIST = 0.1237;  length =    493
  15 vs.  19  DIST = 0.1298;  length =    493
  15 vs.  20  DIST = 0.8519;  length =    493
  15 vs.  21  DIST = 0.3306;  length =    493
  15 vs.  22  DIST = 0.3448;  length =    493
  16 vs.  17  DIST = 0.1501;  length =    493
  16 vs.  18  DIST = 0.1440;  length =    493
  16 vs.  19  DIST = 0.1562;  length =    493
  16 vs.  20  DIST = 0.8621;  length =    493
  16 vs.  21  DIST = 0.3367;  length =    493
  16 vs.  22  DIST = 0.3489;  length =    493
  17 vs.  18  DIST = 0.1197;  length =    493
  17 vs.  19  DIST = 0.1258;  length =    493
  17 vs.  20  DIST = 0.8560;  length =    493
  17 vs.  21  DIST = 0.3347;  length =    493
  17 vs.  22  DIST = 0.3509;  length =    493
  18 vs.  19  DIST = 0.0771;  length =    493
  18 vs.  20  DIST = 0.8560;  length =    493
  18 vs.  21  DIST = 0.3124;  length =    493
  18 vs.  22  DIST = 0.3266;  length =    493
  19 vs.  20  DIST = 0.8621;  length =    493
  19 vs.  21  DIST = 0.3124;  length =    493
  19 vs.  22  DIST = 0.3266;  length =    493
  20 vs.  21  DIST = 0.8621;  length =    493
  20 vs.  22  DIST = 0.8600;  length =    493
  21 vs.  22  DIST = 0.0751;  length =    493


                        Neighbor-joining Method

 Saitou, N. and Nei, M. (1987) The Neighbor-joining Method:
 A New Method for Reconstructing Phylogenetic Trees.
 Mol. Biol. Evol., 4(4), 406-425


 This is an UNROOTED tree

 Numbers in parentheses are branch lengths


 Cycle   1     =  SEQ:   2 (  0.00923) joins  SEQ:   3 (  0.00903)

 Cycle   2     =  SEQ:   1 (  0.01228) joins Node:   2 (  0.00395)

 Cycle   3     = Node:   1 (  0.01631) joins  SEQ:   4 (  0.02933)

 Cycle   4     =  SEQ:   6 (  0.00990) joins  SEQ:   7 (  0.01038)

 Cycle   5     =  SEQ:   8 (  0.00108) joins  SEQ:  10 (  0.00095)

 Cycle   6     = Node:   8 (  0.00828) joins  SEQ:   9 (  0.00794)

 Cycle   7     =  SEQ:  12 (  0.00963) joins  SEQ:  13 (  0.00862)

 Cycle   8     =  SEQ:  11 (  0.02683) joins Node:  12 (  0.01780)

 Cycle   9     = Node:   8 (  0.14056) joins Node:  11 (  0.12466)

 Cycle  10     = Node:   6 (  0.15289) joins Node:   8 (  0.00836)

 Cycle  11     = Node:   6 (  0.27279) joins  SEQ:  20 (  0.43119)

 Cycle  12     = Node:   1 (  0.37750) joins Node:   6 (  0.02113)

 Cycle  13     =  SEQ:  21 (  0.03096) joins  SEQ:  22 (  0.04409)

 Cycle  14     = Node:   1 (  0.19247) joins  SEQ:   5 (  0.19851)

 Cycle  15     = Node:   1 (  0.06705) joins Node:  21 (  0.15555)

 Cycle  16     =  SEQ:  14 (  0.02813) joins  SEQ:  15 (  0.03881)

 Cycle  17     =  SEQ:  18 (  0.03482) joins  SEQ:  19 (  0.04226)

 Cycle  18     = Node:   1 (  0.06938) joins  SEQ:  16 (  0.08032)

 Cycle  19     = Node:   1 (  0.00657) joins Node:  18 (  0.02029)

 Cycle  20 (Last cycle, trichotomy):

                 Node:   1 (  0.00304) joins
                 Node:  14 (  0.02520) joins
                  SEQ:  17 (  0.06202)
(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$ cat primatesAA.dnd
(
(
(
(
(
(
(
(
Human:0.01194,
Chimp:0.01038)
:0.00326,
Gorilla:0.00891)
:0.01097,
Orangutan:0.03264)
:0.00455,
Gibbon:0.06162)
:0.02206,
(
(
(
(
(
PMarmoset:0.02786,
Tamarin:0.03894)
:0.02502,
Owl:0.06203)
:0.00254,
Squirrel:0.08501)
:0.00550,
(
Titi:0.03790,
Saki:0.03902)
:0.01803)
:0.00215,
(
Howler:0.08024,
(
Spider:0.03647,
Woolly:0.04031)
:0.02397)
:0.01410)
:0.16210)
:0.04786,
(
Colobus:0.00868,
DLangur:0.00950)
:0.00712)
:0.00639,
Patas:0.02804)
:0.00658,
(
Rhes_cDNA:0.01224,
Baboon:0.00788)
:0.00441,
(
(
AGM:0.00195,
Tant_cDNA:0.00000)
:0.00770,
AGM_cDNA:0.00783)
:0.01098);
(base) suki@LAPTOP-5PMP3V9O:~/phylo-class$